app/templates/project-name.Gulp/tasks/browser-sync.js

```diff
@@ -1,15 +1,17 @@
 import browserSync from 'browser-sync';
-import gutil from 'gulp-util';
+import colors from 'ansi-colors';
 import config from '../config';
 
+const BROWSERSYNC_OPTIONS = {
```

Comment:
Just to be clear on when we use `UPPER_SNAKE_CASE`, and trust me I'm not trying to be annoying 😄

When I use it:

* When I need to have something like `enum` (e.g. https://redux.js.org/basics/actions, https://vuex.vuejs.org/en/mutations.html#using-constants-for-mutation-types, https://github.com/kraftvaerk/generator-rammevaerk/blob/master/app/index.js#L10). Altough we don't have `enum`s in JavaScript, sometimes it's really needed, like you'd have in other languages:
```
enum Planet {
  case mercury
  case venus
  case earth
}
```
* When I have something simple and "set in stone" (e.g. `const APP_LANGUAGE = 'da';`), as it kind of similar to what you normally have in `.env` files or Bash.

For usual object instance I'd go with the `camelCase`, but I'm looking forward to hear from you 👍

Reply:
I agree to that.. I guess for me it just felt like those options were set in stone 🌮

Reply:
Gotcha, now I know why 👍